### PR TITLE
[JENKINS-68042] Help button alignment is wrong if no object is present to place it on (regression in 2.320)

### PR DESCRIPTION
**Pull request by Langer0416**
See [JENKINS-68042](https://issues.jenkins.io/browse/JENKINS-68042).

Notification URL before:
Notification URL before

Notification URL after:
Notification URL after

Session Termination before:
Session Termination before

Session Termination after:
Session Termination after

### Proposed changelog entries

* Fix the position of the help button when it is not directly attached to an object (regression in 2.320).

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] (If applicable) Jira issue is well described
- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [X] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

### CODE REVIEW

**Comment by Langer0416** [comment]
I have made some changes.

Notification URL
Notification URL after

Session Termination
Session Termination after

File: core/src/main/resources/lib/form/entry.jelly
Comment (Langer0416) [reply core/src/main/resources/lib/form/entry.jelly]: I have made some changes and it won't affect the "Post Steps" section.

Post Steps

**Review by NotMyFault — APPROVED** [review]
Thanks for addressing the outstanding concerns. Buttons suffering from the same regression in other locations are looking fine now again as well.

**Review by basil — APPROVED** [review]
This PR is now ready for merge. We will merge it after approximately 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process. Thanks!

**Pull request merged**
